app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,19 @@
+class TasksController < ApplicationController
+  def create
+    @user = User.find(params[:user_id])
+    @task = @user.tasks.create(task_params)
+    redirect_to root_path
+  end
+
+  def destroy
+    @user = User.find(params[:user_id])
+    @task = @user.tasks.find(params[:id])
```

Comment:
Do we need user to find task of id params[:id]? Can't we directly get task with params[:id]